### PR TITLE
editor translate

### PR DESCRIPTION
**Pull request by stechu**
RT.

### CODE REVIEW

**Comment by hanxiaomax** [comment]
@stechu any updates?

**Comment by stechu** [comment]
@hanxiaomax  working on it.

**Comment by hanxiaomax** [comment]
Please make sure you can finish it this week, can you ?  or you can hand over to others.

from next week, others can translate this file.

thank you in advance

**Comment by stechu** [comment]
@hanxiaomax should be ok. will let you know if I cannot finish.

**Comment by stechu** [comment]
@hanxiaomax  done.

File: _2020/editors.md [hunk 1]
```diff
@@ -220,94 +191,96 @@ def main():
...
-- [nerdtree](https://github.com/scrooloose/nerdtree): file explorer
-- [vim-easymotion](https://github.com/easymotion/vim-easymotion): magic motions
+- [ctrlp.vim](https://github.com/ctrlpvim/ctrlp.vim): 模糊文件查找
+- [ack.vim](https://github.com/mileszs/ack.vim): 代码搜索
+- [nerdtree](https://github.com/scrooloose/nerdtree): 文件浏览器
+- [vim-easymotion](https://github.com/easymotion/vim-easymotion): 魔术操作
 
 We're trying to avoid giving an overwhelmingly long list of plugins here. You
 can check out the instructors' dotfiles
+我们尽量避免在这里提供一长串插件。 你可以查看授课人们的点文件
```
Comment (hanxiaomax) [inline _2020/editors.md:267]:
```suggestion
我们尽量避免在这里提供一份冗长的插件列表。你可以直接查看讲师们开源的配置文件
```

File: _2020/editors.md [hunk 2]
```diff
@@ -316,124 +289,124 @@ If you're a Bash user, use `set -o vi`. If you use Zsh, `bindkey -v`. For Fish,
...
+Readline](https://tiswww.case.edu/php/chet/readline/rltop.html) 库来作为
+它们的命令控制行界面。 Readline 也支持基本的 Vim 模式，
+可以通过在 `~/.inputrc` 添加如下行开启：
 
 ```
 set editing-mode vi
 ```
 
-With this setting, for example, the Python REPL will support Vim bindings.
+在这个设置下， 比如， Python REPL 会支持 Vim 快捷键。
```
Comment (hanxiaomax) [inline _2020/editors.md:308]: 插入语翻译的时候最好拿到前面或者后面，比较符合中文表达习惯
```suggestion
比如，在这个设置下，Python REPL 会支持 Vim 快捷键。
```

File: _2020/editors.md [hunk 3]
```diff
@@ -316,124 +289,124 @@ If you're a Bash user, use `set -o vi`. If you use Zsh, `bindkey -v`. For Fish,
...
 
 ## 其他
 
 There are even vim keybinding extensions for web
-[browsers](http://vim.wikia.com/wiki/Vim_key_bindings_for_web_browsers), some
-popular ones are
+甚至有 Vim 的网页浏览键盘绑定扩展
```
Comment (hanxiaomax) [inline _2020/editors.md:313]:
```suggestion
甚至有 Vim 的网页浏览快捷键
```

key binding 全部统一翻译为快捷键

File: _2020/editors.md [hunk 4]
```diff
@@ -316,124 +289,124 @@ If you're a Bash user, use `set -o vi`. If you use Zsh, `bindkey -v`. For Fish,
...
 - `%s/\[.*\](\(.*\))/\1/g`
-    - replace named Markdown links with plain URLs
+    - 将有命名的 Markdown 链接替换成简单 URLs
 
 ## 多窗口
 
-- `:sp` / `:vsp` to split windows
-- Can have multiple views of the same buffer.
+- 用 `:sp` / `:vsp` 来分割窗口 
+- 能有一个缓存的多个视角。
```
Comment (hanxiaomax) [inline _2020/editors.md:339]:
```suggestion
- 同一个缓存可以在多个窗口中显示
```

File: _2020/editors.md [hunk 5]
```diff
@@ -316,124 +289,124 @@ If you're a Bash user, use `set -o vi`. If you use Zsh, `bindkey -v`. For Fish,
...
-- `{number}@{character}` executes a macro {number} times
-- Macros can be recursive
-    - first clear the macro with `q{character}q`
-    - record the macro, with `@{character}` to invoke the macro recursively
-    (will be a no-op until recording is complete)
-- Example: convert xml to json ([file](/2020/files/example-data.xml))
-    - Array of objects with keys "name" / "email"
-    - Use a Python program?
-    - Use sed / regexes
+- `q{字符}` 来开始在寄存器 `{字符` 中录制宏
```
Comment (hanxiaomax) [inline _2020/editors.md:343]:
```suggestion
- `q{字符}` 来开始在寄存器 `{字符}` 中录制宏
```

File: _2020/editors.md [hunk 6]
```diff
@@ -316,124 +289,124 @@ If you're a Bash user, use `set -o vi`. If you use Zsh, `bindkey -v`. For Fish,
...
         - `%s/<name>\(.*\)<\/name>/"name": "\1",/g`
         - ...
-    - Vim commands / macros
-        - `Gdd`, `ggdd` delete first and last lines
-        - Macro to format a single element (register `e`)
-            - Go to line with `<name>`
+    - Vim 命令 / 宏
+        - `Gdd`, `ggdd` 删除第一行和最后一行
+        - 格式化最后一个元素的宏 （寄存器 `e`）
+            - 到有 `<name>` 的行
```
Comment (hanxiaomax) [inline _2020/editors.md:363]:
```suggestion
            - 跳转到包含 `<name>` 的行
```

**Comment by stechu** [comment]
@hanxiaomax 
Thanks for the review. Addressed these comments.

**Comment by hanxiaomax** [comment]
cool, thank you

**Pull request merged**